src/em/prov/em_provisioning.cpp

```diff
@@ -265,14 +339,104 @@ int em_provisioning_t::create_bss_config_req_msg(uint8_t *buff)
     tmp += sizeof(em_cmdu_t);
     len += static_cast<unsigned int> (sizeof(em_cmdu_t));
 
-    // One miltiap profile tlv 17.2.47
-    // One supported service tlv 17.2.1
-    // One akm suite cap tlv 17.2.78
-    // One or more ap radio basic capability tlv 17.2.7
-    // zero or more backhaul STA radio capabilities tlv 17.2.65
-    // One profile 2 ap capability tlv 17.2.48
-    // One or more AO radio advanced capabilities tlv 17.2.52
-    // One BSS configuration request tlv 17.2.84
+    // 5.3.8 Fronthaul BSS and Backhaul BSS configuration
+    // If an Enrollee Multi-AP Agent has established a PMK and PTK with the Controller at 1905-layer using the procedures
+    // described in section 5.3.7, it shall request configuration for its fronthaul BSSs and backhaul BSSs by sending a BSS
+    // Configuration Request message to the Controller. The BSS Configuration Request message shall include at least
+
+    //  One Multi-AP Profile TLV.
+    tlv = reinterpret_cast<em_tlv_t *> (tmp);
+    tlv->type = em_tlv_type_profile;
+    tlv->len = htons(sizeof(em_profile_type_t));
+    memcpy(tlv->value, &profile_type, sizeof(em_profile_type_t));
+
+    tmp += sizeof(em_tlv_t) + sizeof(em_profile_type_t);
+    len += sizeof(em_tlv_t) + sizeof(em_profile_type_t);
```

Comment:
Can you switch to using this utility method for adding this and the other TLVs
```suggestion
    tmp = em_msg_t::add_tlv(tmp, &len, em_tlv_type_profile, reinterpret_cast<uint8_t *> (&profile_type), static_cast<unsigned int> (sizeof(em_profile_type_t)));
```